src/votekit/metrics.py

```diff
@@ -0,0 +1,95 @@
+from votekit.cvr_loaders import rank_column_csv
+from votekit.election_types import STV, fractional_transfer
+
+
+fpath = "/Users/emariedelanuez/VoteKit/src/votekit/mn_clean_ballots.csv"
```

Comment:
there should not be hard-coded paths, or global variables like this in the library